share/spice/polls/polls.js

```diff
@@ -0,0 +1,58 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_polls = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error) {
+            return Spice.failed('polls');
+        }
+        
+        var results = api_result;
+
+        if (!results) {
+            return Spice.failed('polls');
+        }
+        
+        if(results.length > 50){
+            results = results.splice(50);
+        }
+
+        // Render the response
+        DDG.require("moment.js", function() {
+        Spice.add({
+            id: "polls",
```

Comment:
Once the idea is created, approved and claimed an IA Page will automatically be created for this Instant Answer. We'll need to ensure the `id` here matches the `id` we assign to the page. `election_polls` should be good :+1: